src/main/java/me/totalfreedom/totalfreedommod/command/Command_smite.java

```diff
@@ -38,22 +39,23 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
             return true;
         }
 
-        smite(player, reason);
+        smite(player, sender, reason);
         return true;
     }
 
-    public static void smite(Player player)
+    public static void smite(Player player, CommandSender sender)
```

Comment:
@savnith, perhaps you misunderstood what Lemon has said. He said other commands/listeners are using this method and needs to be updated.